comms/dht/src/config.rs

```diff
@@ -89,6 +92,10 @@ impl Default for DhtConfig {
             broadcast_cooldown_max_attempts: 3,
             broadcast_cooldown_period: Duration::from_secs(60 * 30),
             discovery_request_timeout: Duration::from_secs(2 * 60),
+            #[cfg(test)]
```

Comment:
Integration tests will set the `network` field to Testnet.

Maybe just a convenience wrapper around
```
DhtConfig {
  network:  Network::LocalTest,
  .. DhtConfig::default()
}
```
is a better approach

Reply:
I've made it use LocalTest for cfg(test) and provided a convenience function for use with integration tests and tests outside of the crate